website/docs/r/synthetics_canary.html.markdown

```diff
@@ -0,0 +1,101 @@
+
+## Example Usage
+
+```hcl
+resource "aws_synthetics_canary" "some" {
+  name                 = "some-canary"
+  artifact_s3_location = "s3://some-bucket/"
+  execution_role_arn   = "some-role"
+  handler              = "exports.handler"
+  zip_file             = "test-fixtures/lambdatest.zip"
+  runtime_version      = "syn-1.0"
+
+  schedule {
+    expression = "rate(0 minute)"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `artifact_s3_location` - (Required) Location in Amazon S3 where Synthetics stores artifacts from the test runs of this canary.
+* `execution_role_arn` - (Required) ARN of the IAM role to be used to run the canary. see [AWS Docs](https://docs.aws.amazon.com/AmazonSynthetics/latest/APIReference/API_CreateCanary.html#API_CreateCanary_RequestSyntax) for permissions needs for IAM Role.
+* `handler` - (Required) Entry point to use for the source code when running the canary. This value must end with the string `.handler` .
+* `name` - (Required) Name for this canary.
+* `runtime_version` - (Required) Specifies the runtime version to use for the canary. Currently, the only valid values are `syn-nodejs-2.0`, `syn-nodejs-2.0-beta`, and `syn-1.0`. For more information about runtime versions, see [Canary Runtime Versions](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_Library.html).
+* `schedule` -  (Required) Configuration block providing how often the canary is to run and when these test runs are to stop. Detailed below.
+
+The following arguments are optional:
+
+* `failure_retention_period` - (Optional) Number of days to retain data about failed runs of this canary. If you omit this field, the default of 31 days is used. The valid range is 1 to 455 days.
+* `run_config` - (Optional) Configuration block for individual canary runs. Detailed below.
+* `s3_bucket` - (Optional) Full bucket name which is used if your canary script is located in S3. The bucket must already exist. Specify the full bucket name including s3:// as the start of the bucket name. **Conflicts with `zip_file`.**
+* `s3_key` - (Optional) S3 key of your script. **Conflicts with `zip_file`.**
+* `s3_version` - (Optional) S3 version ID of your script. **Conflicts with `zip_file`.**
+* `start_canary` - (Optional) Whether to run or stop the canary.
+* `success_retention_period` - (Optional) Number of days to retain data about successful runs of this canary. If you omit this field, the default of 31 days is used. The valid range is 1 to 455 days.
+* `tags` - (Optional) Key-value map of resource tags
+* `vpc_config` - (Optional) Configuration block. Detailed below.
+* `zip_file` - (Optional) ZIP file that contains the script, if you input your canary script directly into the canary instead of referring to an S3 location. It can be up to 5 MB. **Conflicts with `s3_bucket`, `s3_key`, and `s3_version`.**
+
+### schedule
```

Comment:
Since the modern documentation has links on the right side of the page to all the sections, internal links to the subsections aren't needed.